functions/src/default.ts

```diff
@@ -5,6 +5,7 @@ export const appConfig: AppConfig = {
   size: {
     disabled: false,
     maxSizeIncrease: 1000,
+    circleCiStatusName: 'ci/circleci: build',
```

Comment:
I think the name is "ci/circleci: build-packages-dist"

Reply:
That's the Angular one, but the default one for circle ci is this.
Will need an update to the angular/angular config file to use `ci/circleci: build-packages-dist` though

Reply:
ok